server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -217,9 +217,10 @@ public enum MetadataVersion {
     // Add ELR related supports (KIP-966).
     IBP_3_9_IV1(22, "3.9", "IV1", true),
 
-    // Introduce version 1 of the GroupVersion feature (KIP-848).
+    // Introduce version 1 of the GroupVersion feature (KIP-848) and transaction versions 1 and 2 (KIP-890)
```

Comment:
Since group/txn features don't depend on this MV, perhaps changing the comment to sth like the following?

bootstrap metadata version of version 1 of the GroupVersion feature (KIP-848) and transaction versions 1 and 2 (KIP-890)